### PR TITLE
Create taxonomies (#30) and initial categories

### PR DESCRIPTION
**Pull request by tmfrnz**
- Adding global and Samoa specific taxonomies to seed file (#30) 
- Adding initial categories for global taxonomies
- Add attribute 'tags_users' to taxonomy table

### CODE REVIEW

**Comment by tmfrnz** [comment]
@barancw can you have a look at this please? 

Also I was wondering if FactoryGirl will set the missing attributes from here?
https://github.com/dumparkltd/hr-nmrf-server/blob/seed-taxonomies-%2330/spec/factories/categories.rb#L5

If yes, we should remove description and url for categories (as I have set up most categories)

File: db/schema.rb [hunk 1]
```diff
@@ -125,6 +125,7 @@
     t.string   "title",                null: false
     t.boolean  "tags_recommendations"
     t.boolean  "tags_measures"
+    t.boolean  "tags_users"
```
Comment (barancw) [inline db/schema.rb]: @tmfrnz This should be done with a rails database migration.

You'd run:
rails g migration AddTagUsersToTaxonomies tag_users:boolean

File: db/seeds.rb [hunk 2]
```diff
@@ -21,21 +21,433 @@ def run_env_seeds!
...
+        title:'H1 Human rights defenders',
+        short_title:'H1'
+      )
+
+
+      # Affected Persons (http://uhri.ohchr.org/search/annotations)
+      FactoryGirl.create(
+          :category,
+          taxonomy:persons,
+          title:'Children',
```
Comment (barancw) [inline db/seeds.rb]: @tmfrnz You are right, it would fill in the description.

You can get around that with this
`
description: '',
`

Comment (tmfrnz) [inline db/seeds.rb]: Thanks @barancw, I will add this.

**Pull request merged**